spotlight-client/src/contentApi/sources/us_me.ts

```diff
@@ -0,0 +1,685 @@
+          metricTypeId: "ParolePopulationCurrent",
+        },
+        {
+          title: "How has the parole population changed over time?",
+          body:
+            "Broadly speaking, increased activity in earlier parts of the criminal justice system (such as arrests and sentencing) will result in increases in the parole population. Changes in parole sentence lengths, earlier releases from prison, etc. may also contribute to the rise and fall of this number.",
+          metricTypeId: "ParolePopulationHistorical",
+        },
+        {
+          title: "What happens after parole?",
+          body:
+            "After parole, a person may be successfully discharged or revoked to prison. Take a look at how the rate of successful parole completion has changed over time, and how the overall rate of successful parole completion varies by demographic.",
+          metricTypeId: "ParoleSuccessHistorical",
+        },
+        {
+          title: "Why do revocations happen?",
+          body:
+            "Revocations happen when a person on parole violates a condition of their supervision or commits a new crime. In Maine, parole revocations fall into one of three categories: technical violation, new offense, and absconsion.",
+          metricTypeId: "ParoleRevocationsAggregate",
+        },
+        {
+          title: "Free Through Recovery program",
+          body:
+            '<a href="https://www.behavioralhealth.nd.gov/addiction/FTR-old" >Free Through Recovery (FTR)</a> is a community based behavioral health program designed to increase recovery support services to individuals involved with the criminal justice system who have behavioral health concerns. This map shows the number of people enrolled in the FTR program today.',
+          metricTypeId: "ParoleProgrammingCurrent",
+        },
+      ],
+    },
+  },
+  localities: {
+    Sentencing: {
+      label: "Judicial District",
+      entries: judicialDistricts,
+    },
+    Prison: {
+      label: "Facility",
+      entries: [
+        { id: "ALL", label: "All Facilities" },
+        { id: "BTC", label: "Bismarck Transition Center" },
+        { id: "DWCRC", label: "Dakota Women's Correctional" },
+        { id: "FTPFAR", label: "Fargo-Female Transition Program" },
+        { id: "MTPFAR", label: "Fargo-Male Transition Program" },
+        { id: "GFC", label: "Grand Forks County Correctional" },
+        { id: "JRCC", label: "James River Correctional Center" },
+        { id: "LRRP", label: "Lake Region Residential Reentry Center" },
+        { id: "FTPMND", label: "Mandan-Female Transition Program" },
+        { id: "MTPMND", label: "Mandan-Male Transition Program" },
+        { id: "MRCC", label: "Missouri River Correctional" },
+        { id: "NDSP", label: "Maine State Penitentiary" },
+        { id: "TRC", label: "Tompkins Rehabilitation And Corrections Center" },
```

Comment:
Instead of committing bogus localities here and elsewhere I would just leave these out and only include the ALL entry for now